pkg/util/ipvs/ipvs.go

```diff
@@ -134,6 +135,7 @@ func GetRequiredIPVSModules(kernelVersion *version.Version) []string {
 }
 
 // IsRsGracefulTerminationNeeded returns true if protocol requires graceful termination for the stale connections
-func IsRsGracefulTerminationNeeded(proto string) bool {
-	return !strings.EqualFold(proto, "UDP") && !strings.EqualFold(proto, "SCTP")
+// Note: Need to be case sensitive for Protocol.
+func IsRsGracefulTerminationNeeded(proto v1.Protocol) bool {
+	return !strings.EqualFold(string(proto), string(v1.ProtocolUDP)) && !strings.EqualFold(string(proto), string(v1.ProtocolSCTP))
```

Comment:
If you are going to use `v1.Protocol` then you should only match the actual `v1.Protocol` values. If you need to be case-insensitive, then you shouldn't use `v1.Protocol`.